index.html

```
@@ -1,685 +1,697 @@
-  container: '#share-button',
-  url: 'https://your-website.com',
-  title: 'Check this out!',
-  description: 'An amazing website'
-});</code>
+      <!-- Quick Start -->
+      <div class="demo-section">
+        <h2>🚀 Quick Start</h2>
+        <p>Get started in seconds with minimal setup</p>
+
+        <div class="code-block">
+          <button class="copy-btn" aria-label="Copy code">Copy</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)